lib/ui.ex

```diff
@@ -75,13 +75,14 @@ defmodule Bootleg.UI do
   """
   def puts_upload(%SSHKit.Context{} = context, local_path, remote_path) do
     Enum.each(context.hosts, fn(host) ->
-      [:bright, :green]
+      [:reset, :bright, :green]
         ++ ["[" <> String.pad_trailing(host.name, 10) <> "] "]
         ++ [:reset, :yellow, "UPLOAD", " "]
         ++ [:reset, Path.relative_to_cwd(local_path)]
         ++ [:reset, :yellow, " -> "]
         ++ [:reset, Path.join(context.path, remote_path)]
-      |> Bunt.puts()
+      |> IO.ANSI.format(coloring_enabled())
+      |> IO.puts()
```

Comment:
Should this also be `IO.binwrite`? `puts_download` and `puts_send` have the same question.